src/DoocsBackend.cc

```diff
@@ -5,27 +5,32 @@
  *      Author: Martin Hierholzer
  */
 
-#include <algorithm>
-#include <boost/algorithm/string.hpp>
-#include <ChimeraTK/BackendFactory.h>
-#include <ChimeraTK/DeviceAccessVersion.h>
-#include <libxml++/libxml++.h>
-#include <fstream>
+#include "DoocsBackend.h"
 
 #include "CatalogueCache.h"
 #include "CatalogueFetcher.h"
-#include "DoocsBackend.h"
+#include "DoocsBackendEventIdAccessor.h"
 #include "DoocsBackendFloatRegisterAccessor.h"
-#include "DoocsBackendIIIIRegisterAccessor.h"
 #include "DoocsBackendIFFFRegisterAccessor.h"
+#include "DoocsBackendIIIIRegisterAccessor.h"
+#include "DoocsBackendImageRegisterAccessor.h"
 #include "DoocsBackendIntRegisterAccessor.h"
 #include "DoocsBackendLongRegisterAccessor.h"
 #include "DoocsBackendStringRegisterAccessor.h"
-#include "DoocsBackendEventIdAccessor.h"
 #include "DoocsBackendTimeStampAccessor.h"
 #include "RegisterInfo.h"
 #include "StringUtility.h"
 #include "ZMQSubscriptionManager.h"
+#include <libxml++/libxml++.h>
```

Comment:
Do we really need all of these new headers?

Reply:
I now removed libxml header